composer.json

```diff
@@ -7,6 +7,7 @@
         "php": ">=7.1.0",
         "ext-dom": "*",
         "ext-json": "*",
+        "ext-pdo": "*",
```

Comment:
We're already using PDO but this was not declared in the Composer manifest. PHPStorm was complaining about this.